index.js

```diff
@@ -1,5 +1,6 @@
 module.exports = {
     Test: {
-        VlCookieConsent: require('./test/e2e/components/vl-cookie-consent')
+        VlCookieConsent: require('./test/e2e/components/vl-cookie-consent.js'),
+        VlCookieConsentOptIn: require('./test/e2e/components/vl-cookie-consent-opt-in.js')
```

Comment:
heb je die rechtstreeks nodig nu dan?

Reply:
Ik heb de bestanden opgesplitst, leek mij proper om beide aan te bieden?

Reply:
ge gaat normaal altijd wel nen opt-in vastkrijgen via de VlCookieConsent dus niet strikt noodzakelijk denk ik, maar ook niet mega-harmful. natuurlijk, grotere publieke API = meer kans op breaking changes ...

Reply:
Het page object `VlCookieConsent` exposed bij `getOptIns` `VlCookieConsentOptIn` elementen, dus het is zelfs nodig.

Reply:
Nee dat is niet waar denk ik, daar kan eender welk object met de juiste API uit komen. If it quacks like a duck ... leve de non-getypeerdheid van JS. 
Maar zoals gezegd, niet heel erg lijkt me.

Reply:
Nodig is relatief ja, het is properder in mijn ogen om te exposen wat allemaal uit de API vloeit. Al was het maar om code completion te hebben in je IDE.

Reply:
Ja, da snijdt dus langs 2 kanten, want nu kunt ge da intern bvb niet hernoemen als ge da zou willen ... maar zoals gezegd, hier niet zo van belang, maar in het algemeen, hoe groter uw publieke API hoe meer ge moet (blijven) ondersteunen naar uw afnemers toe

Reply:
De publieke API van `VlCookieConsentOptIn` kan je toch nooit zomaar aanpassen? Als je de eend niet meer laat kwaken, breekt het sowieso bij de afnemers ook al gebruiken ze `VlCookieConsentOptIn` niet 'proper'?

Reply:
Nee, maar je kan wel intern het bestand vl-cookie-consent-opt-in.js hernoemen naar vl-cookie-consent-flopt-in.js als je dat zou willen

Reply:
Dat kan je nu toch ook? Dan pas je de index.js aan en afnemers merken daar niets van...